### ISSUE TITLE
GWAS result files in Wiki

### ISSUE THREAD

**Issue by scx18927** [JonJala/mtag#142]
Hi, I'm the beginner in genetic analysis.
I'm trying to follow the steps of  tutorial in Wiki section, but example files you offer seem to be broken.
I'm wondering if there are other analogous files, may be smaller in size, you could offer for me to follow the tutorial and to familiar with the format of input files.

Thanks for taking time reading my issues.

**Comment by JonJala** [comment]
Hi, are there files in particular that aren't working for you / appear to
be broken?  What issues are you noticing with them, and how far do you get
in the tutorial before you have problems?

**Comment by scx18927** [comment]
I'm sorry. I've solved the problem. Here is the question i met with.

Originally, i downloaded the files you offer:
![螢幕擷取畫面 2021-09-27 154241](https://user-images.githubusercontent.com/91283848/134865626-ae0e84cb-4b8c-4950-9ad7-181038c87b11.png)

I found it is a .gz file, and the file could not be open. therefore, I thought the file might be broken.
After rereading the tutorial thoroughly, I found the input file is a .txt file.
I changed (deleted) the file extension and it works

Thank you, I really appreciate your reply.
